### ISSUE TITLE
Documentation is so minimal and hard to follow

### ISSUE THREAD

**Issue by RidenShark** [django-notifications/django-notifications#293]
Hi. I am a student with intermediate experience in Django. I've been trying to implement the notification system for almost a day now and I haven't got past the point of showing live notifications and linking them to the default template area.

The documentation is insufficient for me as I find it hard to make sense of what is happening without an example or code.

For example, the documentation says what to put in the template, but nothing about how to mark messages unread, or how to override the default templates. It does say to use API calls to mark read messages unread, but not how.

I'd contribute to the wiki if I know how to do all this, but I need help. Can someone link me to a proper way to use this app (perhaps a GitHub project that uses it in a simple way (I checked the test app, but it's not easy to follow in the context of setting up either))? Once I understand, maybe I can contribute to the project wiki.

This is a very nice project and I very much love to use it in my apps, learn, and contribute. Please help me.

**Comment by RidenShark** [comment]
Any updates?

**Comment by qrkourier** [comment]
I'm new to Django and struggling to follow the instructions to get started with django-notifications. After installing the PyPi module, adding to `INSTALLED_APPS`, and adding `import` and `urlpatterns` in the second-level site's URLConf in urls.py I'm getting this error when I attempt `migrate` or `runserver`.

> NameError: name 'url' is not defined

**Comment by backtrackbaba** [comment]
@RidenShark Check out this [GitHub search query](https://github.com/search?l=Text&q=%22django-notifications%22&type=Code) to find the projects that have mentioned django-notifications in their requirements.txt file

Hope this helps!

**Comment by RidenShark** [comment]
@backtrackbaba, it helps a bit, but definitely not as helpful as clear and detailed documentation. I looked at some of the repos and it still is not clear to me how to implement it.

**Comment by nemesisdesign** [comment]
**This software is provided for free with the hope that it will be useful but without warranty of any kind.**

If you want to contribute to make it better you're welcome to do so.

I personally do not have the time to do more than what I can. I hope you guys understand we maintainers also have our own life and responsibilities, we help out and provide our work for free to you all.

I am closing this because it's not something actionable that proposes anything in particular.

If you have precise proposals for improvement, please open another issue.

**Comment by RidenShark** [comment]
Hey @nemesisdesign, I do understand the value of your contributions, and that too for free and open. I am just pointing out that it's not welcoming for beginner-level users like me. From your position, as you guys are experienced, maybe it might appear most of the things are self-explanatory and doesn't need documentation.

This could be easily adapted into a lot of projects if the documentation is something as beautiful as Django's very own documentation. At least that's what I feel. I tried thrice implementing it so far and I couldn't. If I could, I would've documented it myself.

I really hope someone else who had implemented it would contribute.
